automation/utils.py

```diff
@@ -80,13 +90,11 @@ def get_planner_command(directory=None):
 def get_receiver_command(directory=None):
     return _get_command(directory, "receiver.py")
 
-DEFAULT_CONFIG = 'aos_at: 0\nlocation:\n  elevation: 138\n  latitude: 54.9\n  longitude: 54.9\n  name: Taneczna\nmax_elevation_greater_than: 0\nnorad:\n- https://celestrak.com/NORAD/elements/noaa.txt\n- https://celestrak.com/NORAD/elements/active.txt\nsatellites:\n- freq: 137.62e6\n  name: NOAA 15\n- freq: 137.9125e6\n  name: NOAA 18\n- freq: 137.1e6\n  name: NOAA 19\nserver:\n  id: 1\n  secret: 873730d540e1154cf3203b09546ac1fe\n  url: http://127.0.0.1:5000/receive\nstrategy: max-elevation\n'
-
 def open_crontab() -> CronTab:
     if DEV_ENVIRONEMT:
         path = os.path.join(CONFIG_DIRECTORY, "cron.tab")
```

Comment:
I've never seen anyone spelling the name that way. It's always crontab (lower case, no dots or other separators).

Reply:
It is only text file for dev use. It needs name and I like extensions. But it is no problem to rename it. ;)